packages/super-components/yarn.lock

```diff
@@ -0,0 +1,4664 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
remove this file, you didn't change anything in this folder

Reply:
Done